FATN_to_USB/tests/conftest.py

```
@@ -0,0 +1,338 @@
+    response.text = 'Internal Server Error'
+    return response
+
+
+# ============================================================================
+# Process/Subprocess Fixtures
+# ============================================================================
+
+@pytest.fixture
+def mock_subprocess_run():
+    """Mock subprocess.run for testing system commands."""
+    def _mock_run(cmd, **kwargs):
```

Ruff (0.14.14)

Unused function argument: `kwargs`

(ARG001)